spec/features/pets/pets_delete_spec.rb

```diff
@@ -51,8 +51,13 @@
       click_link "Approve #{pet_1.name} for Adoption"
     end
 
+
     expect(page).to have_current_path("/pets/#{pet_1.id}")
     expect(page).to_not have_link('Delete Pet')
+    visit "/pets"
+    expect(page).to_not have_link('Delete Pet')
+    visit "/shelters/#{shelter_1.id}/pets"
+    expect(page).to_not have_link('Delete Pet')
```

Comment:
found a bug while i was testing the website, forgot to hide this link in pets index and shelter_pets index